test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -2204,365 +1933,124 @@ BOOST_AUTO_TEST_CASE(function_modifier)
 	ABI_CHECK(callContractFunctionWithValue("getOne()", 1), encodeArgs(1));
```

Comment:
Why is this not extractable?

Reply:
Ah good point! I just found out yesterday how value can be send with `isoltest`. Initially I was just ignoring calls to `callContractFunctionWithValue`. So I guess we can extract more test-cases.

Reply:
great!

Reply:
sadly enabling `callContractFunctionWithValue` just show that one test-case more can be extracted.. a lot of those test-cases use `m_evmHost`, `m_contractAddress` or `balanceAt`..